default.qdbd.cfg

```diff
@@ -4,6 +4,9 @@
             "async_ts": {
                 "flush_deadline": 500
             }
+        },
+        "network": {
+            "profile_performance": false
```

Comment:
But... that's the default already, nope? So what's the point?

Reply:
Hahaha hum yes I want to put "true" there